zerver/tests/test_custom_profile_data.py

```diff
@@ -453,6 +453,68 @@ def test_delete_internals(self) -> None:
         self.assertFalse(self.custom_field_exists_in_realm(field.id))
         self.assertEqual(user_profile.customprofilefieldvalue_set.count(), self.original_count - 1)
 
+    def test_delete_value_with_editable_by_user(self) -> None:
+        iago = self.example_user("iago")
+        hamlet = self.example_user("hamlet")
+        realm = iago.realm
+
+        biography = CustomProfileField.objects.get(name="Biography", realm=realm)
+        birthday = CustomProfileField.objects.get(name="Birthday", realm=realm)
+
+        # set editable_by_user to false
+        self.login_user(iago)
+        data = {}
+        data["name"] = birthday.name
```

Comment:
Is this name field required in this API query?

Reply:
No, this is no longer required. I have removed this field.